Makefile

```diff
@@ -16,8 +16,8 @@ install: ## Install dependencies locally
 build: ## Build the container
 	docker build --target ${REAL_ENV} --tag ${TAG} .
 
-start: ## Start the container
-	docker run --detach --name ${NAME} --publish 8080:8080 ${MOUNT} ${TAG}
+restart: ## If the container's running, stop it, then start the container
+	make stop && docker run --detach --name ${NAME} --publish 8080:8080 ${MOUNT} ${TAG}
```

Comment:
Make can have dependencies like:

```make
command: dependency ## Comment
```

Suggest keeping `start` as is, and have a separate `restart` command that runs `stop` then `start`.